client/src/app/parlamentar/parlamentar.component.html

```diff
@@ -78,9 +78,27 @@
       <div class="col">
         <div class="parlamentar-title">{{ parlamentar.nomeEleitoral }}</div>
         <p>
-          {{getCargoParlamentar(parlamentar?.casa)}}
-          <br>
-          {{ parlamentar.parlamentarPartido.sigla }}/{{ parlamentar.uf }}
+          {{getCargoParlamentar(parlamentar?.casa)}} - {{ parlamentar.parlamentarPartido.sigla }}/{{ parlamentar.uf }}
+        </p>
+        <p>
+          <b>Data de nascimento:</b>
+          {{ parlamentar.dataNascimento | date:'dd-MM-yyyy' }}
```

Comment:
```suggestion
          {{ parlamentar.dataNascimento | date:'dd/MM/yyyy' }}
```
Troca traços por barras na data de nascimento.